# This message contains information on the main electrical currents drawn from the EPS
float64 b					# battery
float64 fl 					# front left wheel
float64 rl 					# rear left wheel
float64 rr 					# rear right wheel
float64 fr 					# front right wheel
